packages/features/form-builder/Components.tsx

```diff
@@ -398,17 +408,24 @@ export const Components: Record<FieldType, Component> = {
 
       const { t } = useLocale();
 
-      const getCleanLabel = (option: { label: string; value: string }): string | JSX.Element => {
-        if (!option.label) {
+      const didUserProvideLabel = (
```

Comment:
This is who we decide if the organizer has changed the label and if the label has been changed by the organizer, we show the organizer's label instead of the first option label.